### ISSUE TITLE
Only last statement in SQL editor executes

### ISSUE THREAD

**Issue by Grismar** [pawelsalawa/sqlitestudio#4256]
### Details
If you write a query consisting of multiple statements, only the last statement appears to execute, even though the entire query is parsed without error (and errors are caught in prior statements)

### Steps to reproduce
Write a two line query, i.e.:
```
insert into data values(0);
insert into data values(1);
```
Run the query, note that only one row is affected and only the last line has an apparent effect on the table.

### Operating system
Windows 10 Pro 64-bit.

### SQLiteStudio version
v3.2.1

**Comment by Grismar** [comment]
Thanks for getting back to me - this was indeed the issue. To me, the default is counter-intuitive to me - obviously, hence the question, but apologies for not finding this in the documentation myself. I changed it as you suggested, but I do appreciate how it might be useful and I like that it is configurable.

Although perfection doesn't exist, given that this seems like a non-obvious feature (it clearly is that), perhaps it would be good to provide a non-intrusive hint on-screen that this is happening and can be configured? (one that can be dismissed forever after first showing it, ideally?) Or, alternatively, perhaps have a toggle button on the SQL editor to quickly switch between `execute selected line / execute all lines` would be a great added feature? (I would have found it, as I did look around for an "execute all button", I just didn't take the step of looking at configuration)
